src/main/java/com/example/pressarticle/article/service/ArticleService.java

```diff
@@ -0,0 +1,62 @@
+package com.example.pressarticle.article.service;
+
+import com.example.pressarticle.article.model.Article;
+import com.example.pressarticle.article.repository.ArticleRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+@Service
+@Slf4j
+public class ArticleService {
+
+    private ArticleRepository articleRepository;
+
+    public ArticleService(ArticleRepository articleRepository) {
+        this.articleRepository = articleRepository;
+    }
+
+    public List<Article> sortAllArticleTransfer(
+            Integer pageNumber,
+            Integer pageSize,
+            String sortBy
+    ) {
+        Pageable pageable = PageRequest.of(pageNumber, pageSize, Sort.by(sortBy).descending());
+        Page<Article> quotations = articleRepository.findAll(pageable);
+        if (quotations.hasContent()) {
+            return quotations.getContent();
+        } else {
+            return Collections.emptyList();
+        }
+    }
+
+    public List<Article> articleIdTransfer(Long id) {
+        return articleRepository.findArticleById(id);
+    }
+
+    public List<Article> articleDescribeTransfer(String text, String titleText) {
+        return articleRepository.findArticleByDescribe(text, titleText);
+    }
+
+    public Article addArticleTransfer(Article article) {
+        return articleRepository.save(article);
+    }
+
+    public Article updateArticleTransfer(
+            Long id,
+            Article toUpdate
+    ) {
+        toUpdate.setId(id);
```

Comment:
To podejście IMHO jest bardzo złe - nie powinno się zmieniać id encji zarządzanej przez Hibernate, bo to może całkowicie zdemolować cache pierwszego poziomu. Używanie settera dla id powinno być wręcz zakazane (niestety nie można go wyrzucić bo chyba jest używany przez Jacksona przy deserializacji, ale można spróbować)
Standardowo robi się to tak:

wyciągamy z bazy artykuł po danym id
przepisujemy do niego te pola które chcemy zaktualizować (oczywiście poza id !)
zapisujemy do bazy (teoretycznie nawet i tego nie trzeba bo się zapisze samo przy zamknięciu sesji, ale możemy chcieć zwrócić zapisany obiekt, tak więc ok)